nuxeo-features/nuxeo-annotation/nuxeo-annotation-core/src/main/resources/OSGI-INF/AnnotationService.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
Let's name this file `annotation-service.xml` instead of `AnnotationService.xml`